src/pages/gen2/start/quickstart/vite-react-app/index.mdx

```diff
@@ -0,0 +1,273 @@
+export const meta = {
+  title: 'Vite + React App',
+  description: 'Get started with AWS Amplify (Gen 2) using React + Vite.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Vite, and React. If you are new to these technologies, we recommend you go through the [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html), [Vite](https://vitejs.dev/guide/), and official [React](https://react.dev/learn/tutorial-tic-tac-toe) tutorials first.
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+
+<Fragments fragments={{ javascript: prerequisites, nextjs: prerequisites }} />
+
+## Create a project
+
+From your project directory, run the following command and respond to the prompts:
+
+```bash title="Terminal" showLineNumbers={false}
```

Comment:
@chrisbonifacio When I copy this via the icon, I get more than just the command `npm create vite@latest`.

![image](https://github.com/aws-amplify/docs/assets/21967/456ba111-8544-4465-aab7-0c3f3fbfae64)

Reply:
Looks like this when I continue and would be confusing for a beginner

![image](https://github.com/aws-amplify/docs/assets/21967/00bca21d-0300-4402-956e-305fbe58b02c)

Reply:
My suggestion is to use a single command that uses their React + TS template via https://vitejs.dev/guide/#scaffolding-your-first-vite-project

`npm create vite@latest react-amplify-gen2 -- --template react-ts`

Reply:
Change the command to the suggestion and tested locally to confirm. Thanks!

Also, corrected the mention of the directory name in the next paragraph